OSD4/1-prereqs.md

```diff
@@ -7,11 +7,11 @@ You will need to [download and install](https://docs.openshift.com/dedicated/cli
 
 ![CLI Tools](images/0-cli_tools_page.png)
 
-##### Why use `oc` over `kubectl`
+**Why use `oc` over `kubectl`**<br>
```

Comment:
Is the `<br>` valid here?

Reply:
I added that because for some reason it was all appearing on the same line in local testing.